upgrade/Stage_0_Prerequisites.md

```diff
@@ -451,8 +454,11 @@ example showing how to find the IUF activity.
    Save the name of the CFS configurations:
 
    ```bash
-   export KUBERNETES_CFS_CONFIG_NAME="management-22.4.0-csm-x.y.z"
-   export STORAGE_CFS_CONFIG_NAME="storage-22.4.0-csm-x.y.z"
+   export KUBERNETES_CFS_CONFIG_NAME=""
+   ```
+
+   ```bash
+   export STORAGE_CFS_CONFIG_NAME=""
    ```
 
    Note that the storage node configuration might also be titled `minimal-management` depending on the value
```

Comment:
Remove or edit this note since the example CFS name was removed from above.

Reply:
My run on surtur had this prefix, should I still remove it?

Reply:
I think you could leave it and just add that it could have the 'storage' prefix as well. 
```suggestion
   Note that the storage node configuration might be titled `minimal-management-` or `storage-` depending on the value
```